internal/services/appservice/linux_function_app_data_source_test.go

```diff
@@ -24,6 +25,37 @@ func TestAccLinuxFunctionAppDataSource_standardComplete(t *testing.T) {
 	})
 }
 
+func TestAccLinuxFunctionAppDataSource_basicOutputs(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_linux_function_app", "test")
+	d := LinuxFunctionAppDataSource{}
+
+	ipListRegex := regexp.MustCompile(`(([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})(,){0,1})+`)
+
+	data.DataSourceTest(t, []acceptance.TestStep{
+		{
+			Config: d.basic(data, SkuBasicPlan),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).Key("outbound_ip_addresses").MatchesRegex(ipListRegex),
+				check.That(data.ResourceName).Key("outbound_ip_address_list.#").Exists(),
+				check.That(data.ResourceName).Key("possible_outbound_ip_addresses").MatchesRegex(ipListRegex),
+				check.That(data.ResourceName).Key("possible_outbound_ip_address_list.#").Exists(),
+				check.That(data.ResourceName).Key("default_hostname").MatchesRegex(regexp.MustCompile(`(.)+`)),
```

Comment:
I think it would be fine just to add these conditions to the `standardComplete` test above since they should pass in all circumstances?

Reply:
I added these test cases to standardComplete (same for the Windows data source test)